templates/robotics/what-is-ros.html

```diff
@@ -69,31 +68,30 @@ <h2>What is ROS?</h2>
   </div>
 </section>
 
-<section class="p-strip is-bordered">
-  <div class="row u-equal-height">
-    <div class="col-4 u-vertically-center u-hide--small">
+<section class="p-strip">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-5 u-embedded-media u-hide--small">
+        <iframe title="Getting started with ROS" class="u-embedded-media__element" src="https://www.youtube.com/embed/videoseries?list=PL_2PosskAdC0lfoecAPKBV4GJ0oZv7W_X" allowfullscreen=""></iframe>
+    </div>
+    <div class="col-6 col-start-large-7">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/7dad0dcc-logo-ros2.svg",
+          url="https://assets.ubuntu.com/v1/9b32c9a4-3979232.png",
           alt="",
-          height="240",
-          width="240",
+          height="50",
+          width="50",
           hi_def=True,
           attrs={"class": ""},
           loading="lazy",
         ) | safe
       }}
-    </div>
-    <div class="col-8">
       <h2>What is ROS 2?</h2>
       <p>ROS 2 is a complete re-design of the framework, tackling the shortcomings of the first generation, effectively bringing it to industry needs and standards. ROS 2 contains new sets of packages that can be installed alongside ROS 1 to ease migration to a more secure platform. ROS 2 takes advantage of new technologies and newly updated APIs asked for by the community.</p>
-      <p><a class="p-button--positive p-link--external" href="https://www.ros.org/contribute/">Contribute to ROS 2</a></p>
-      <p><a class="p-link--external" href="https://index.ros.org/doc/ros2/">Learn more about ROS 2 from the owners</a></p>
-    </div>
+      <p><a class="p-button--positive p-link--external" href="https://www.ros.org/contribute/">Get started with ROS 2</a></p>
```

Comment:
The doc says the url should point here: https://index.ros.org/doc/ros2/Tutorials/